{"filename": "haskell-enclosed-exceptions-1.0.3-333-x86_64.pkg.tar.zst", "name": "haskell-enclosed-exceptions", "base": "haskell-enclosed-exceptions", "version": "1.0.3-333", "desc": "Catching all exceptions from within an enclosed computation", "csize": "13507", "isize": "40181", "md5sum": "ce4682a9f86500c1b2fa98d94ec3d868", "sha256sum": "2f9c46414f708056c582fab3016df5c0320399e6aa68cbed680dc3202d7de0fc", "pgpsig": "iQIzBAABCgAdFiEEtZcfLFwQqaCMYAMPeGxj8zDXy5IFAmjJPT0ACgkQeGxj8zDXy5LIqQ/9HEb6r4U7X0WF6F2NbxfjM0JAyGyOCC+ozKtDX25bWpWRwoBD7w0JIffb6c17z70fwBW9RB0B9MOmb/wyw0zF127gEmGv+n3hYGMmlLb8DT4EYlpaX7ARmdRNsIkW1tnVTuYmHN+wSQOBGqASZ8y0dsrLtXlv1Ut9cszW0/PE3m7Vk2ltgU4eqFOgT3BxYlpOPwXtQ8JYqg9qzhQx3Ypg43w8ZQDPLXzOdXvUHirxqV3uZhmTSU/Q2mVXJe2h8ZYQydy+5suA5MzkjTmVH9+7oT7yP2a1cbbE8TH3FKHOdAiCZ3PcSBMHgylDxJMN9XPqXUFBPkFar3qNxfgx6tnr0XwnJP1x3cVYVb/KVffLOMoi5RN/f+cZ90ijShimIc/InPClNdtCO70m9qkkOoPYtR4T9gzP6WMWyiPAjDlIXyZZMP8kR2LBwU9cxfkdgLWH1fFLZIBR49koAvAQurqsBGTBMFIbk283qC+ce0O44+hFStwfbzE5urfgC6xcn5Bub4KEIjZiNgiUCUNYPvFtL037+bGi/oYqaO4v/ZjDcIU3v4SONvAmSnbKBg4kq6ps1nJGbstzn4FO0sLxyGTTHgy94CcmxcskVOjBxiR3B6VkP2EeWKx8W7C3gppbKsshRtvXjVQiFFGCOuCwpm48t1WPzoPoLdOMQCtWFoelRYc=", "url": "https://github.com/jcristovao/enclosed-exceptions", "license": "MIT", "arch": "x86_64", "builddate": "1758018851", "packager": "Felix Yan <felixonmars@archlinux.org>", "depends": ["ghc-libs", "haskell-lifted-base", "haskell-monad-control", "haskell-transformers-base"], "makedepends": ["ghc"]}